Documentation/gettingstarted/encryption.rst

```diff
@@ -17,6 +17,12 @@ as Cilium managed host traffic, will be encrypted using IPsec. This guide uses
 Kubernetes secrets to distribute keys. Alternatively, keys may be manually
 distributed, but that is not shown here.
 
+.. note::
+
+    The Helm examples in this document operate on the ``kube-system`` namespace,
+    whereas other Getting Started Guides utilize the ``cilium`` namespace. Make
+    sure to deploy ``Secret`` resources and Cilium in the same namespace!
```

Comment:
I think it would be better to avoid the reference to other GSGs, since we might end up changing them at some point.
How about something like: "The secret resources need to be deployed on the same namespace as Cilium (in our example we use ``kube-system``)".